ouroboros-consensus/test-consensus/Test/Dynamic/RealPBFT.hs

```diff
@@ -128,8 +128,7 @@ mkProtocolRealPBFT (NumCoreNodes n) (CoreNodeId i)
           dlgKey
           dlgCert
 
-    signatureThreshold = PBftSignatureThreshold $
-      1.0 / (fromIntegral n + 1.0)
+    signatureThreshold = PBftSignatureThreshold $ n `seq` 2.0
```

Comment:
TODO: we should think about whether we can expect consensus for a given threshold + `TestConfig`.

Reply:
Should the threshold vary as the number of nodes in the network changes?

Reply:
I read the PBFT spec to better understand, and it occurred to me that this shouldn't prevent consensus. So I've pushed a reversion of this commit, `8c09cc6a - TEMPORARY inflate signatureThreshold in RealPBFT`.

The insight is that `8c09cc6a` only seemed like it prevented a test case failure because it preceded `d2a9dbed - consensus: introduce InvalidRollForward exception`. Once `d2a9dbed` and `5291db9c - test-consensus: restart CS client on InvalidRollForward` are included, `8c09cc6a` is no longer necessary.

The `determineExpectations` function is unaware that PBFT + late joins means some of the earlier joiners might not be able to forge blocks in their leadership slots. However, I think that particular inaccuracy of `determineExpectations` might not be relevant: it's accurate enough to catch the 1-block-chain corner-case (since that's too early for PBFT to reject the blocks) and it's accurate enough once the last node joins the chain -- so it always returns the right expectation for the test's final chains. I anticipate that its ignorance of PBFT block rejection might become a problem once we have other failures, such as network partitions.